siriuspy/siriuspy/timesys/csdev.py

```diff
@@ -511,23 +511,30 @@ def get_hl_trigger_database(hl_trigger, prefix=''):
     dbase['Src-Sts'] = _dcopy(dic_)
     dbase['Src-Sel'] = dic_
 
+    max_dur = 17e6
+    max_wid_raw = 2123400000
```

Comment:
`max_wid_raw` can be 2147483648 = 2^31 - 1

Reply:
thanks. done!